app/components/ClientSampleCode.tsx

```diff
@@ -116,5 +117,37 @@ export function ClientSampleCode({
       break
   }
 
-  return <Highlight language={language} code={code} />
+  return (
+    <>
+      <div>
+        {language == 'python' ? (
+          <div>
+            Run this command to install with{' '}
+            <Link href="https://pip.pypa.io/">pip</Link>:
+            <pre>
+              <code className="hljs language-sh">pip install gcn-kafka</code>
+            </pre>
+            or this command to install with with{' '}
+            <Link href="https://docs.conda.io/">conda</Link> :
+            <pre>
+              <code className="hljs language-sh">
+                conda install -c conda-forge gcn-kafka
+              </code>
+            </pre>
+          </div>
+        ) : null}
+        {language == 'mjs' ? (
+          <div>
+            Run this command to install with{' '}
+            <Link href="https://www.npmjs.com">npm</Link>:
+            <pre>
+              <code className="hljs language-sh">npm install gcn-kafka</code>
+            </pre>
+          </div>
+        ) : null}
+        Sample code:
+      </div>
```

Comment:
Can this stuff go in the `switch` statement above please?

Reply:
Moved it into the switch and use a .map in the body of the component